WrappedPlugin/index.js

```diff
@@ -149,7 +149,9 @@ const wrapHooks = (orig, pluginName, smp, type) => {
   };
 
   const wrapped = Object.keys(hooks).reduce((acc, method) => {
-    acc[method] = genProxy(method);
+    if (method != "normalModuleLoader")
```

Comment:
it is too hardcoded, now it is arguing with `additionalChunkAssets`
so I changed it to be like 
```
if (hooks[method])
  acc[method] = genProxy(method);
```